cmd/massa-wallet/massa-wallet.go

```diff
@@ -27,7 +27,12 @@ func StartServer(app *walletApp.WalletApp) {
 		promptApp = prompt.NewEnvPrompter(app)
 	}
 
-	AssetsStore, err := assets.NewAssetsStore()
+	assetsJSONPath, err := assets.GetAssetsJSONPath()
+	if err != nil {
+		logger.Warnf("Failed to get AssetsStore JSON file: %v", err)
```

Comment:
have you tested this? what happens when assetsJSONPath is not found at line 35 `assets.NewAssetsStore(assetsJSONPath)`

Reply:
GetAssetsJSONPath simply calls AccountPath() which  create the directory if it doesn't exist;